### PR TITLE
webplotdigitizer: deprecate 

### PR DESCRIPTION
**Pull request by khipp**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

The development of WebPlotDigitizer has been stopped (see [announcement](https://automeris.io/v4/)).

### CODE REVIEW

**Comment by bevanjkay** [comment]
It doesn't look like it is discontinued, but has moved to a closed source build. Is there any way to update the cask to version 5?

**Comment by khipp** [comment]
@bevanjkay There is only a web-based version currently.

From the [Version 5 announcement](https://automeris.io/posts/):

> Offline Desktop Version: Version 5 will not have a offline desktop version for a while. This can be resumed once features are more stable. It also may/may not be feasible to run AI assist features in a 100% offline manner.

**Comment by krehel** [comment]
Please squash commits to simplify - thank you!

**Comment by khipp** [comment]
Left the deprecation in a separate commit, in case we still want to drop that.

File: Casks/w/webplotdigitizer.rb [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
-  url "https://automeris.io/downloads/WebPlotDigitizer-#{version}-darwin-x64.zip"
+  url "https://apps.automeris.io/downloads/WebPlotDigitizer-#{version}-#{arch}.zip"
   name "WebPlotDigitizer"
   desc "Extract numerical data from plot images"
   homepage "https://automeris.io/WebPlotDigitizer.html"
 
-  livecheck do
-    url "https://automeris.io/download.html"
-    regex(%r{href=.*?/WebPlotDigitizer-(\d+(?:\.\d+)*)-darwin-x64\.zip}i)
+  deprecate! date: "2023-06-10", because: :discontinued
```
Comment (Bo98) [inline Casks/w/webplotdigitizer.rb:13]:
```suggestion
  deprecate! date: "2024-06-10", because: :discontinued
```

**Review by miccal — APPROVED** [review]

**Pull request merged**